src/test_workflow/smoke_test/smoke_test_cluster_opensearch.py

```diff
@@ -47,6 +46,8 @@ def __init__(
         self.dist = self.bundle_manifest.build.distribution
         self.distribution = Distributions.get_distribution(self.product, self.dist, self.version, work_dir)
 
+        self.test_recorder = test_recorder
```

Comment:
Why do we move this line?

Reply:
Revert change in the latest commit. Thanks!